Nightwatch.Resources/Shell.fs

```diff
@@ -10,9 +10,12 @@ open Nightwatch.Core.Resources
 
 let private create (processController : Process.Controller)
                    (param : IDictionary<string, string>) =
-    let command = param.["cmd"]
+    let commandParams = param.["cmd"].Split(' ')
+    let len = commandParams.Length
+    let command = commandParams.[0]
+    let args = Array.sub commandParams 1 (len - 1)
```

Comment:
Probably will be irrelevant after the fix, but you should remember there's `Array.tail`.

Reply:
Oh, thanks!